spec/appliance_console/logfile_configuration_spec.rb

```diff
@@ -89,22 +90,47 @@
       EOT
     end
 
-    it "stops and starts evm and configures the logfile disk" do
-      subject.new_logrotate_count = 3
-      subject.disk = double(@spec_name, :size => "9999999", :path => "fake disk")
+    context "evm was running" do
```

Comment:
If there's only one spec in the `context` I think you can just add the context description at the end of the spec name. In this case it would be:
`it "stops and starts evm and configures the logfile disk when evm was running" do`